frontend/common/APICall.js

```diff
@@ -41,10 +50,18 @@ class APICall{
         };
 
         return new Promise((resolve, reject) => {
-            fetch(Links.serverBaseURL + `/${path}`, options)
+            fetch(`${Links.serverBaseURL}/${path}`, options)
             .then(response => {
                 if (!response.ok) {
-                    throw new Error('Network response was not ok');
+                    if(response.status === 401){
+                        localStorage.setItem('signedIn', "false");
+                        window.location.replace('index.html');
+                    } else if(response.status === 404){
+                        throw new Error('Not found!!');//should redirect to 404 page
+                    } else {
+                        throw new Error('Network response was not ok');
```

Comment:
we throw an error that network response was not okay but why ? the message from the response must also be in the error message

Reply:
Does the api actually returns a message of what went wrong?
Because I can't access the message property
![image](https://github.com/Spiderman-Trivia-Inc/Spiderman-Trivia-Inc/assets/159114894/5d123aa1-7705-4ae3-b3ff-2e1f0abffedb)

Reply:
It does return what went wrong on the response.message 
Try logging out the response and see what it returns